.gitignore

```diff
@@ -28,3 +28,5 @@ rust/Cargo.lock
 rust/target
 rust/src/clib/test/nmstate_test
 rust/src/clib/test/vgcore*
+
+*.swp
```

Comment:
It seems unrelated and specific to Vim. Please, put this on a different commit.

Reply:
Removed.